templates/bootstrap/plugin_name/app/serializers.py.j2

```diff
@@ -72,7 +72,7 @@ class {{ plugin_camel_short }}RemoteSerializer(platform.RemoteSerializer):
         model = models.{{ plugin_camel_short }}Remote
 
 
-class {{ plugin_camel_short }}RepoairoeySerializer(platform.RepoairoeySerializer):
```

Comment:
Probably my fingers got misplaced on the keys

Reply:
Hopefully you can get some rest tomorrow after we branch 3.0 :)